src/app/organisms/emoji-board/EmojiBoard.jsx

```diff
@@ -147,8 +150,9 @@ function EmojiBoard({ onSelect, searchRef }) {
   function selectEmoji(e) {
     if (isTargetNotEmoji(e.target)) return;
 
-    const emoji = e.target;
-    onSelect(getEmojiDataFromTarget(emoji));
+    const emoji = getEmojiDataFromTarget(e.target);
+    onSelect(emoji);
+    addRecentEmoji(emoji.unicode);
```

Comment:
we should not add custom emoji to recent for now as they are not used by wide variety of client. also we are updating `io.element.recent_emoji` which doesn't support custom emoji.